tests

```diff
@@ -259,14 +265,64 @@ EOF
 	./safe get secret/delete/six:wom; exitok $? 1
 	./safe get secret/delete/six:beep; exitok $? 0
 
-	testing $version attempt to delete non-existent key
-	./safe delete secret/delete/gobbledegook; exitok $? 0
+	testing $version attempt to delete non-existent key should fail
+	./safe delete secret/delete/gobbledegook; exitok $? 1
 
-	testing $version attempt to delete non-existing key within existing secret should not error
+	testing $version attempt to delete non-existing key within existing secret should error
 	./safe set secret/delete/seven foo=bar
-	./safe delete secret/delete/seven:gobbledegook; exitok $? 0
+	./safe delete secret/delete/seven:gobbledegook; exitok $? 1
 	./safe get secret/delete/seven:foo; exitok $? 0
 
+	testing $version delete non-existent key should not fail if -f is given
+	./safe delete secret/delete/gobbledegook -f; exitok $? 0
+
+	testing $version attempt to delete non-existing key within existing secret should error
+	./safe set secret/delete/eight foo=bar
+	./safe delete secret/delete/eight:gobbledegook -f; exitok $? 0
+	./safe get secret/delete/seven:foo; exitok $? 0
```

Comment:
Just a messaging thing that I fixed elsewhere and forgot to here. It's really that the operation should fail silently with a -f.